lua/roslyn/slnutils.lua

```diff
@@ -0,0 +1,68 @@
+---@param file_paths string[]
```

Comment:
Could you add a return param for the methods in this file, and also explicitly do `return nil` where you somewhere just do a `return`?

Reply:
done